week-3/alarmclock/index.html

```diff
@@ -4,16 +4,24 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Title here</title>
+    <title>Alarm Clock App</title>
   </head>
   <body>
     <div class="centre">
-      <h1 id="timeRemaining">Time Remaining: 00:00</h1>
-      <label for="alarmSet">Set time to:</label>
+        <div class="progress-bar" id="circularProgressBar">
+          <div class="progress-bar-inner">
+            <div id="timeRemaining">00:00</div>
+          </div>
+        </div>
+     
+      <label for="alarmSet"></label>
       <input id="alarmSet" type="number" />
```

Comment:
The unit of the input is not clear, whether it is minutes or seconds. How could you make it clearer for the user to know that the input is supposed to be in seconds?